### PR TITLE
Adaptive Bubble Sort Implemented Issue #2325 fixed

### PR DESCRIPTION
**Pull request by SaloniSwagata**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Making the bubble sort adaptive so that it runs only till the array is unsorted. I have also changed the name of the array from 'list' to 'Array' because in Python, 'list' is a keyword. Also in the initial code, the final list was not printed, so changed that too.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#2325

### CODE REVIEW

**Review by HarshCasper — APPROVED** [review]

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Few changes I would recommend: 

- Use Comments above the Line of Code 
- Create a function which can be called while using the Code 
- Enclose the driver code in a `if__name__ == "__main__":` block of code

**Comment by SaloniSwagata** [comment]
> Few changes I would recommend:
> 
> * Use Comments above the Line of Code
> * Create a function which can be called while using the Code
> * Enclose the driver code in a `if__name__ == "__main__":` block of code

Okay, I'll fix them

**Comment by SaloniSwagata** [comment]
@HarshCasper I've made the required changes.

File: Python/sort/Bubble_Sort.py [hunk 1]
```diff
@@ -1,7 +1,28 @@
...
-print("List after sorting: ", list);         #Printing out the sorted list
+'''
+bubble_sort() function uses an iterative approach sort the array in increasing
+order passed as an argument. It compares adjacent elements and swappes them if
+there is any larger element before the smaller element. After the first pass,
+the most heaviest element shifts to the last position.
+The flag is used to check if any swapping is performed or not. If no swapping is
+done, the loop breaks as the array is already sorted.
+Time Complexity: O(n^2)
+Space Complexity: O(1)
```
Comment (HarshCasper) [inline Python/sort/Bubble_Sort.py]: Shift Time and Space Complexity at the end of the Code

File: Python/sort/Bubble_Sort.py [hunk 2]
```diff
@@ -1,7 +1,28 @@
...
+        if flag:
+            break
+
+if __name__=="__main__": 
+    print("Enter the elements of the list to be sorted: ")
+    Array = [int(x) for x in input().split(" ")]
+    bubble_sort(Array)
+    print("List after sorting: ")
+    for i in Array:
+        print(i,end=" ")
```
Comment (HarshCasper) [inline Python/sort/Bubble_Sort.py]: Add sample I/O

**Review by HarshCasper — CHANGES_REQUESTED** [review]

**Comment by SaloniSwagata** [comment]
@HarshCasper I've reordered the time and space complexity, added sample I/O and also fixed some bugs which I found.

**Comment by siddharth25pandey** [comment]
@SaloniSwagata Add the update in **Readme** file

**Comment by SaloniSwagata** [comment]
@siddharth25pandey I just edited an existing code. The code is already linked in the readme file.

**Comment by siddharth25pandey** [comment]
> @siddharth25pandey I just edited an existing code. The code is already linked in the readme file.

@SaloniSwagata Alright then :)

**Review by siddharth25pandey — APPROVED** [review]
@SaloniSwagata LGTM, able to merge

**Comment by siddharth25pandey** [comment]
@all-contributors please add @SaloniSwagata for her code

**Pull request merged**